yarn-project/ethereum/src/deploy_l1_contracts.ts

```diff
@@ -176,13 +180,25 @@ export const deployL1Contracts = async (
     logger(`Deployed DecoderHelper at ${decoderHelperAddress}`);
   }
 
+  let headerDecoderHelperAddress: EthAddress | undefined;
+  if (contractsToDeploy.headerDecoderHelper) {
+    headerDecoderHelperAddress = await deployL1Contract(
+      walletClient,
+      publicClient,
+      contractsToDeploy.headerDecoderHelper.contractAbi,
+      contractsToDeploy.headerDecoderHelper.contractBytecode,
+    );
+    logger(`Deployed HeaderDecoderHelper at ${headerDecoderHelperAddress}`);
```

Comment:
Similar to earlier. If feels a bit funky that we are having these things getting into not-test area. But since the full deployment is essentially something that is not expected to be part of the "node" package I guess it don't make a big diff here. But having it in the archiver seems wrong.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4015/commits/388aa21bf22c876671f98a2868fca63d23a16a05